### ISSUE TITLE
Transitive dependencies scan

### ISSUE THREAD

**Issue by ksastan** [pivotal/LicenseFinder#741]
Hello there!

Didn't find information about possibility to scan transitive dependencies.
For example I have project with several dependencies, this dependencies has their own dependencies, and their dependencies has dependencies ... etc.

Can I get list of all dependencies of dependencies with their licenses? 
Or maybe you solve this problem by another way?

**Comment by xtreme-shane-lattanzio** [comment]
Hey @ksastan! The closest thing I can think of for what you are talking about is the `--recursive` flag but I think that is more for project paths, unfortunately not for deepening the dependency tree. This would be a cool feature to have. We would probably want it to be optional similar to the recursive flag.

**Comment by an2deg** [comment]
Hi @xtreme-shane-lattanzio !

Would it be possible if the implementation be respect ignore groups like devDependency ?

**Comment by xtreme-shane-lattanzio** [comment]
@an2deg Have you tried running license_finder ignored_groups --project-path='#{project_path}' add devDependencies first? Are you saying recursive does not respect this?

**Comment by an2deg** [comment]
Right. I added devDependency to the ignored groups and then executed license_finder with flag  --recursive. 

license_finder skipped dev deps of the main project but not of the project dependencies.

**Comment by xtreme-shane-lattanzio** [comment]
Hmm seems like a bug then. Not sure when we will get to this though so if you want to take a look it, we would get this moving faster. Thanks!
